pkg/clusters/addons/metallb/metallb_test.go

```diff
@@ -1,21 +1,20 @@
-package metallb_test
+package metallb
 
 import (
 	"net"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 
-	ktfmetal "github.com/kong/kubernetes-testing-framework/pkg/metallb"
-	ktfnet "github.com/kong/kubernetes-testing-framework/pkg/networking"
+	ktfnet "github.com/kong/kubernetes-testing-framework/pkg/utils/networking"
```

Comment:
why alias the package name?

https://github.com/golang/go/wiki/CodeReviewComments#imports

> Avoid renaming imports except to avoid a name collision; good package names should not require renaming. [...]

Reply:
removed in 71744f635d29be15db11a7e636478da0667fc44b :+1:

Reply:
for consistency consider renaming the `docker` import too (optional)

Reply:
:+1: resolved in bf6a4ca712d82ae6b0ffba7a0f0457fa1d4b43ea